python/triton/compiler/compiler.py

```diff
@@ -375,6 +378,7 @@ def _init_handles(self):
         # TODO: n_regs, n_spills should be metadata generated when calling `ptxas`
         self.module, self.function, self.n_regs, self.n_spills = driver.active.utils.load_binary(
             self.name, self.kernel, self.metadata.shared, device)
+        self._handles_initialized = True
```

Comment:
Why's this necessary?  Shouldn't the check the `self.module` is populated be enough?

Reply:
Oh, good catch. Checking `self.module` is enough.